CalibCalorimetry/EcalTPGTools/plugins/EcalTPGParamBuilder.cc

```diff
@@ -451,6 +418,69 @@ void EcalTPGParamBuilder::analyze(const edm::Event& evt, const edm::EventSetup&
   ss.str("");
   //modif-alex-27-july-2015 +-Jean june 2016-end
   
+  //modif-alex-30/01/2012   displaced in analyze Jean 2018
+  ss.str("");
+  if(useTransparencyCorr_){
+    if(Transparency_Corr_ != "tag") {       // Jean 2018
+      edm::LogInfo("TopInfo") << "INFO: READING transparency correction files" << "\n";
+      ss << "INFO: READING transparency correction files\n";
+      std::ifstream transparency(Transparency_Corr_.c_str());
+      if(!transparency) edm::LogError("TopInfo") << "ERROR: File " << Transparency_Corr_.c_str() << " could not be opened" << "\n";
+    
+      char buf[1024];
+      transparency.getline(buf,sizeof(buf),'\n');
+      int xtalcounter = 0;
+      while( transparency ) {
+	std::stringstream sin(buf);
+      
+	int raw_xtal_id;
+	sin >> raw_xtal_id;
+      
+	double xtal_trans_corr;
+	sin >> xtal_trans_corr; 
+      
+	//      edm::LogInfo("TopInfo") << raw_xtal_id << " " << xtal_trans_corr << "\n";
+	ss << raw_xtal_id << " " << xtal_trans_corr << "\n";
+      
+	Transparency_Correction_.insert(make_pair(raw_xtal_id,xtal_trans_corr));
+      
+	xtalcounter++;
+	transparency.getline(buf,sizeof(buf),'\n');
+      }//loop transparency
+      transparency.close();
+      ss << "INFO: DONE transparency correction files " << xtalcounter << "\n";
+      edm::LogInfo("TopInfo") << ss.str();
+      ss.str("");
+    }   // file
+    else {       // Jean 2018
```

Comment:
@depasse as far as I understand, this option is so far dummy, and just activated in the test configurationa dded to this PR

Reply:
This is a new feature very convenient to be able to upload payloads
from sqlite files without the need to convert them to xml or txt ones.
   It is neither dummy nor for test only.

Reply:
@depasse I mean that so far there is no configuration activating it in production. Do you confirm?

Reply:
Yes, it is not triggered in production.  It is in EcalTPG"Tools" and will be used in the future ponctually.  It is important to keep the traceability of this.